addon/ng2/blueprints/ng2/files/package.json

```diff
@@ -2,6 +2,9 @@
   "name": "<%= htmlComponentName %>",
   "version": "0.0.0",
   "license": "Apache-2.0",
+  "scripts":{
+    "postinstall": "webdriver-manager update"
```

Comment:
You're right. That's actually the way we're doing.

I thought for a moment that the webdriver version was independent of Protractor, but that doesn't make sense.

We still need to try to make our init time faster. Running `webdriver-manager update` as part of `ng test` would be nice for the first run, but it would be unnecessary for the subsequent runs. What else we can do?